src/client/components/EditChannel/RoomItemEdit.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import fetchWithAuth from '../../utils/fetchWithAuth';
+import './EditRoom.style.css';
+import { Link } from 'react-router-dom';
+
+const RoomItemEdit = (props) => {
+  const [roomTitle, setRoomTitle] = useState('');
+  const channel = props.location.roomItemEditProps[0];
```

Comment:
There is no `location` property that is being passed here. Also, missing PropTypes.

Reply:
In this case location property is accessed from props property.